frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -0,0 +1,117 @@
+                                Save and continue
+                            </Button>
+                        </Form.Item>
+                    </div>
+                    <div>
+                        <Form.Item label="Person selection">
+                            <Form.Item name="filters">
+                                <label>Select the users who will participate in this experiment.</label>
+                                <div style={{ flex: 3, marginRight: 5 }}>
+                                    <PropertyFilters
+                                        endpoint="person"
+                                        pageKey={'1234'}
+                                        onChange={(personProperties) => {
+                                            form.setFieldsValue({ filters: personProperties })
+                                        }}
+                                        propertyFilters={[]}
+                                        // onChange={(properties) => {
+                                        //     onPropertyCriteriaChange({ properties })
+                                        // }}
+                                        // propertyFilters={group.properties}
+                                        style={{ margin: '1rem 0 0' }}
+                                        taxonomicGroupTypes={[
+                                            TaxonomicFilterGroupType.PersonProperties,
+                                            TaxonomicFilterGroupType.CohortsWithAllUsers,
+                                        ]}
+                                        popoverPlacement="top"
+                                        taxonomicPopoverPlacement="auto"
+                                    />
+                                </div>
+                            </Form.Item>
+                        </Form.Item>
+                        <Form.Item className="text-right">
+                            <Button icon={<SaveOutlined />} htmlType="submit" type="primary" onClick={handleNext}>
+                                Save and preview
+                            </Button>
+                        </Form.Item>
+                        <Button onClick={handlePrev}>Go back</Button>
+                    </div>
+                    <div className="confirmation">
+                        <PageHeader title={experiment?.name} />
+                        <div>{experiment?.description}</div>
+                        <div>Owner: {user?.first_name}</div>
+                        <div>Feature flag key: {experiment?.feature_flag}</div>
+                        <Row>
+                            <Col>
+                                <Row>Person allocation</Row>
+                                <Row>The following users will participate in the experiment</Row>
+                                <ul>
+                                    {experiment?.filters?.map((filter: PersonPropertyFilter, idx: number) => (
```

Comment:
Note that: The filters we choose in the "Persons to select" are actually the `properties` attribute of `experiment.filters` .

The rest of the filterType is built by the "funnel selecting" page in the middle, which i guess is missing right now. (We don't want to allow selecting filters there - apply em implicitly, no breakdowns either, just the funnel steps)

If nothing is selected here, it means it applies to All users.

Reply:
do you mean like filters = { properties: [] } ?

Reply:
yes, precisely. (So effectively, the form element is populating the properties of filters for experiment, vs filters itself)